Header header
int32 channels
int32 samples
int32 sample_rate
uint16[] data # C1 C2 C3 C4 C1 C2 C3 C4 ...
